R/checks.R

```diff
@@ -23,23 +24,21 @@ check_user_rights <- function(
     db_data,
     db_metadata
 ) {
-  # Similar to link_arms, use pivot wider to create list elements with top
-  # level form_name and vector of associated field_names
+
   missing_db_metadata <- db_metadata %>%
     filter(!.data$field_name_updated %in% names(db_data)) %>%
-    select("field_name_updated", "form_name") %>%
-    pivot_wider(names_from = "form_name",
-                values_from = "field_name_updated",
-                values_fn = list)
-
-  # Supply user with warning message(s) displaying missing form name and
-  # associated fields
-  for (i in seq_len(ncol(missing_db_metadata))) {
-    warning(paste0(
-      "Form name {", names(missing_db_metadata)[i], "} detected in metadata, but not found in the database export. This can happen when the user privileges are not set to allow exporting that form via the API. The following variables are affected: ",
-      paste(unlist(missing_db_metadata[i][[1]]), collapse = ", ")),
-      call. = FALSE)
-  }
+    select(.data$field_name_updated, .data$form_name) %>%
```

Comment:
I think we're no longer using `.data`?

Reply:
Correct, removed in most recent commit. Also had to update some tests with the new warning classes.